lib/pymedphys/docs/release-notes.md

```diff
@@ -8,6 +8,87 @@ This project adheres to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
+## [0.35.0]
+
+### News around this release
+
+* Installation on MacOS (Intel) has been simplified and is now the same as for other platforms, thanks to work by the new pymssql maintainers.
```

Comment:
Move to docs section? I think reserve this news section for the sort of thing we'd be keen on posting to Linkedin and the like. Not sure if there's anything like that for this release?

Reply:
If I were a Mac user, I'd be interested in knowing I don't have to do a bunch of things other than `pip install pymedphys[user]`

What we might want to do is have a section on installation.
Changing the documentation on installation is documentation.
Changing how one performs the installation is not documentation, it's installation.

But it's a question of what you consider "Newsworthy".

There isn't anything in this release that you want to talk about (e.g. the binary) that is going to be of interest to the casual observer, unless that casual observer is a MacOS user who is thinking of maybe installing PyMedPhys.

For someone who wants to know "should I upgrade", that usually involves reading more deeply (does it fix a bug I'm having issues with, does it improve performance for something that matters to me, is it going to break my current code).

Having a binary is a game changer for many, so when that comes out, it should be in the News.

Reply:
Yes certainly. The binary is something I'd be happy to be in the news. Once it's ready :). Any section that is empty can just be removed. A presentation at a conference, a paper, a new contributor (or new maintainer 🎉🙂) would all count for this section in my opinion.

Reply:
> What we might want to do is have a section on installation.

More than happy to have that added in 🙂